test/unfurl-tests.js

```diff
@@ -1,3 +1,27 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
File first commited in 2021

Reply:
I think it'd be correct to change it to 2021 then.